feature1/one-more-file.txt

```diff
@@ -0,0 +1,3 @@
+This was also added in the feature1 branch.
```

Comment:
It is really good that you have decided to add this line in the one-more-file.txt file

Reply:
Thanks for your comment, dwchub!